src/Console/Commands/OpenApiUiAssetsCommand.php

```diff
@@ -0,0 +1,130 @@
+{
+    /**
+     * File System
+     * @var \Quantum\Libraries\Storage\FileSystem
+     */
+    protected $fs;
+
+    /**
+     * Command name
+     * @var string
+     */
+    protected $name = 'install:openapi';
+
+    /**
+     * Command description
+     * @var string
+     */
+    protected $description = 'Generates files for OpenApi UI';
+
+    /**
+     * Command arguments
+     * @var \string[][]
+     */
+    protected $args = [
+        ['module', 'required', 'The module name'],
+    ];
+
+    /**
+     * Command help text
+     * @var string
+     */
+    protected $help = 'The command will publish OpenApi UI resources';
+
+    /**
+     * Path to public debug bar resources
+     * @var string 
+     */
+    private $publicOpenApiFolderPath = 'public/assets/OpenApiUi';
+
+    /**
+     * Path to vendor debug bar resources
+     * @var string 
+     */
+    private $vendorOpenApiFolderPath = 'vendor/swagger-api/swagger-ui/dist';
+
+    /**
+     * Exclude File Names
+     * @var array
+     */
+    private $excludeFileNames = ['index.html', 'swagger-initializer.js', 'favicon-16x16.png', 'favicon-32x32.png'];
+
+    /**
+     * Executes the command and publishes the debug bar assets
+     */
+    public function exec()
+    {
+        $this->fs = Di::get(FileSystem::class);
+        $module = $this->getArgument('module');
+        $modulePath = modules_dir() . DS . $module;
+        $file = $modulePath . DS . 'Config' . DS . 'routes.php';
+        $openapiRoutes = 'return function ($route) {
+    //$route->group("openapi", function ($route) {
+        $route->get("' . strtolower($module) . '/documentation", function (Quantum\Http\Response $response) {
+            $response->html(partial("openapi/openapi"));
+        });
+
+        $route->get("' . strtolower($module) . '/docs", function (Quantum\Http\Response $response) {
+            $fs = Quantum\Di\Di::get(Quantum\Libraries\Storage\FileSystem::class);
+            $response->json((array) json_decode($fs->get(modules_dir()' . DS . $module . DS . 'Resources' . DS . 'openapi' . DS . 'docs.json", true)));
+        //});
+    });';
+
+        if (!$this->fs->isDirectory($modulePath)) {
+            $this->error('The module ' . $module . ' not found');
+            return;
+        }
+
+        if (!$this->fs->exists(assets_dir() . DS . 'OpenApiUi' . DS . 'index.css')) {
+            $dir = opendir($this->vendorOpenApiFolderPath);
+
+            if (is_resource($dir)) {
+                while (($fileUi = readdir($dir))) {
+                    if ($fileUi && ($fileUi != '.') && ($fileUi != '..') && !in_array($fileUi, $this->excludeFileNames)) {
+                        copy($this->vendorOpenApiFolderPath . DS . $fileUi, $this->publicOpenApiFolderPath . DS . $fileUi);
+                    }
+                }
+
+                closedir($dir);
+            }
+        }
+
+        if (strpos($this->fs->get($file), '$route->group("openapi", function ($route) {') === false) {

```

Comment:
Instead of this we can load all routes and then use route_group_exists() helper method to check if the group exists or not